diffkemp/semdiff/function_diff.py

```diff
@@ -159,7 +159,7 @@ def functions_semdiff(first, second, fun_first, fun_second, config):
 
 def functions_diff(mod_first, mod_second,
                    fun_first, fun_second,
-                   glob_var, config,
+                   glob_var, config, glob_var_value=None,
```

Comment:
```suggestion
                   config,
                   glob_var, glob_var_value=None,
```
This params order would make more sense.